main/webapp/modules/core/scripts/reconciliation/standard-service-panel.js

```diff
@@ -337,7 +337,6 @@ ReconStandardServicePanel.prototype.start = function() {
         identifierSpace: this._service.identifierSpace,
         schemaSpace: this._service.schemaSpace,
         type: (type) ? { id: type.id, name: type.name } : null,
-        autoMatch: JSON.parse(Refine.getPreference("ui.reconciliation.automatch", true)),
```

Comment:
@ayushrai206 : I see you removed your change, but you alos removed all of the original code here…

Reply:
The original code set the default value to true..which i already did in the line i added,so i removed the previous code,also regarding the line i added is it working perfectly?is the variable i pointed the default value to correct?